src/dune/args.py

```diff
@@ -44,105 +44,104 @@ class arg_parser:
+        self._parser.add_argument('--ctest', nargs=1, metavar="<DIR>",
+                                  help='[-- FLAGS] run the ctest tests for a smart contract project at the '
                                        'directory given. Optional flags are of the form -- -VV')
-        self._parser.add_argument('--gdb', nargs=1, metavar=["PROGRAM", "-- FLAGS (Optional)"],
-                                  help='start gdb into the container with given executive binary'
+        self._parser.add_argument('--gdb', nargs=1, metavar="<PROGRAM>",
+                                  help='[-- FLAGS] start gdb into the container with given executive binary'
                                        'Optional flags are of the form -- -VV')
-        self._parser.add_argument('--deploy', nargs=2, metavar=["DIR", "ACCOUNT"],
+        self._parser.add_argument('--deploy', nargs=2, metavar=("<DIR>", "<ACCOUNT>"),
                                   help='deploy a smart contract and ABI to account given')
         self._parser.add_argument('--destroy-container', action='store_true',
-                                  help='destroy context container <Warning, this will destroy '
-                                       'your state and block log>')
+                                  help='destroy context container. Warning! This will destroy '
+                                       'your state and block log')
         self._parser.add_argument('--stop-container', action='store_true',
                                   help='stop the context container')
         self._parser.add_argument('--start-container', action='store_true',
                                   help='start the context container')
-        self._parser.add_argument('--set-core-contract', metavar="ACCOUNT",
+        self._parser.add_argument('--set-core-contract', metavar="<ACCOUNT>",
                                   help='set the core contract to an account given (default '
                                        'normally is `eosio`)')
-        self._parser.add_argument('--set-bios-contract', metavar="ACCOUNT",
+        self._parser.add_argument('--set-bios-contract', metavar="<ACCOUNT>",
                                   help='set the bios contract to an account given (default '
                                        'normally is `eosio`)')
-        self._parser.add_argument('--set-token-contract', metavar="ACCOUNT",
+        self._parser.add_argument('--set-token-contract', metavar="<ACCOUNT>",
                                   help='set the token contract to an account given (default '
                                        'normally is`eosio.token`)')
         self._parser.add_argument('--bootstrap-system', action='store_true',
                                   help='install boot contract to eosio and activate all protocol '
                                        'features')
         self._parser.add_argument('--bootstrap-system-full',
```

Comment:
If I removed '\b' I get:
```
  --bootstrap-system-full [ [ ...]]
                        [CURRENCY] [MAX_VALUE] [INITIAL_VALUE] The same as ...
```
and I think it looks a bit better

Reply:
> do you know where does closing square brackets come from?
> 
> ```
>  --bootstrap-system-full   ...]]
> ```

I guess it looks in the argparse code like print("[{} ...]".format(metavar)) . 
metavar='\b' removes the first bracket. 
With metavar='' it looks like --bootstrap-system-full  [ ...]] . It seems I will change metavar='\b' to metavar='' IMHO it looks even better. Need to check with many parameters.

Reply:
Yes, let's do that. Additionally I checked that in those case where you used '\b' we can remove nargs completely. This way for code:
```
        self._parser.add_argument('--bootstrap-system-full',
                                  metavar='',
                                  help='[CURRENCY] [MAX_VALUE] [INITIAL_VALUE] '
                                       'The same as `--bootstrap-system` but also creates accounts '
                                       'needed for core contract and deploys core, token, '
                                       'and multisig contracts. If optional arguments are provided '
                                       'it creates specific CURRENCY (default "SYS") with maximum amount of '
                                       'MAX_VALUE and initial value of INITIAL_VALUE')
```
we will have:
```
  --bootstrap-system-full 
                        [CURRENCY] [MAX_VALUE] [INITIAL_VALUE] The same as `--bootstrap-system` but also creates accounts needed for
                        core contract and deploys core, token, and multisig contracts. If optional arguments are provided it creates
                        specific CURRENCY (default "SYS") with maximum amount of MAX_VALUE and initial value of INITIAL_VALUE

```

Reply:
I'd prefer to leave metavar='' because the brackets and dots show that the option has parameters --add-app  [ ...] 
Especially it is important in the usage section.

Reply:
with current changes:
```
  --bootstrap-system-full [ [ ...]]
                        [CURRENCY] [MAX_VALUE] [INITIAL_VALUE] The same as `--bootstrap-system` but also creates accounts needed for
                        core contract and deploys core, token, and multisig contracts. If optional arguments are provided it creates
                        specific CURRENCY (default "SYS") with maximum amount of MAX_VALUE and initial value of INITIAL_VALUE
```

I think without `nargs` it looks better, because there is no `[ [ ...]]` which might be confusing.

Reply:
But in your case you can't see in the usage section which options have parameters.